test/test_loader.py

```diff
@@ -0,0 +1,20 @@
+import xypath.loader
+from nose.tools import assert_equal
+fn = "fixtures/4132.zip"
+
+tables = xypath.loader.table_set(fn)
+
+four, = xypath.loader.get_sheets(tables, 0)
```

Comment:
Am assuming the order in the zip file is deliberate:

```
bat:fixtures francis$ unzip -l 4132.zip 
Archive:  4132.zip
  Length     Date   Time    Name
 --------    ----   ----    ----
        5  01-16-15 11:36   four.csv
        4  01-16-15 11:36   one.csv
        6  01-16-15 11:36   three.csv
        4  01-16-15 11:36   two.csv
 --------                   -------
       19                   4 files
```

Might be worth noting in a comment